e2e/cypress/integration/system_console/plugin_upload_spec.js

```diff
@@ -108,6 +110,64 @@ describe('Draw Plugin - Upload', () => {
         cy.findByText(/Installed Plugins/).scrollIntoView().should('be.visible');
         cy.findByTestId('com.mattermost.draw-plugin').should('not.exist');
     });
+
+    it('M27802-MT40-Plugin remains enabled when upgraded', () => {
```

Comment:
The Jira ticket number won't be necessary.
```suggestion
    it('MT40-Plugin remains enabled when upgraded', () => {
```

Reply:
done.